web-common/src/features/connectors/olap/createModel.ts

```diff
@@ -75,16 +77,22 @@ export async function createModelFromTable(
   const topComments =
     "-- Model SQL\n-- Reference documentation: https://docs.rilldata.com/reference/project-files/models";
   const connectorLine = `-- @connector: ${connector}`;
-  const selectStatement = `select * from ${sufficientlyQualifiedTableName}`;
+  const selectStatement = hasSpaces(sufficientlyQualifiedTableName)
```

Comment:
Why not always surround with quotes? Would be convenient when the name is changed later to include spaces.

Reply:
Fair, it's subjective but I think it's a little cleaner to omit the quotes if possible